graphql/resolvers.go

```diff
@@ -934,6 +949,14 @@ func (r *taskResolver) PatchNumber(ctx context.Context, obj *restModel.APITask)
 	return &patch.PatchNumber, nil
 }
 
+func (r *taskResolver) FailedTestCount(ctx context.Context, obj *restModel.APITask) (int, error) {
+	failedTestCount, err := r.sc.GetTestCountByTaskIdAndFilters(*obj.Id, "", []string{evergreen.TestFailedStatus}, obj.Execution)
+	if err != nil {
+		return 0, InternalServerError.Send(ctx, fmt.Sprintf("Error getting tests for failedTestCount: %s", err.Error()))
```

Comment:
is 0 the right return value if error occurs? in the ui, when the patch is loading or if there is an error, i show `- / -` to indicate "we don't know how many tasks there are yet. i think this approach is better than `0 / 0` because those numbers are technically not accurate.

Reply:
ah, i now see that this is the FailedTestCount resolver. in that case 0 does seem like the right return value.